Cookbook/Technical-Documents/LabelsInPRs.md

```diff
@@ -0,0 +1,118 @@
+A product vertical corresponds to a big feature of the product. 
+
+- <span style="background-color:#2BBBBB; color:black; font-weight:600">Health Management 🍄</span> - any work done in the health management vertical
+- <span style="background-color:#5319e7; color:white; font-weight:600">Healthcheck 👩‍⚕️</span> - any work done in the healthcheck vertical
+- <span style="background-color:#25fcb8; color:black; font-weight:600">Payments 💰</span> - any work done in the payments vertical
+- <span style="background-color:#d3cb34; color:black; font-weight:600">Prescriptions 💊</span> - any work done in the prescriptions vertical
+- <span style="background-color:#f9d0c4; color:black; font-weight:600">SDK 💸</span> - any work done in the SDK vertical
+- <span style="background-color:#006b75; color:white; font-weight:600">Triage UI 🏥</span> - any work done in the Triage UI vertical
+
+### <a name="project-horizontals"></a>Project horizontals
+A project horizontal corresponds to an area of the project that has an horizontal impact, meaning it has impact in most or all the verticals in the project.
+
+- <span style="background-color:#d4c5f9; color:black; font-weight:600">Automation Tests 🤖</span> - introduction of automation tests
+- <span style="background-color:#fcfc50; color:black; font-weight:600">Localisation 🌐</span> - updating localizable strings or working in the localization system of the project
+- <span style="background-color:#8631bf; color:white; font-weight:600">Infrastructure 🛠</span> - working in the foundation of the project like network layer.
+- <span style="background-color:#2fcbe0; color:black; font-weight:600">Tooling 🔨</span> - working in tools like CircleCI, Danger, Linter.
+
+### <a name="branding"></a>Branding
+We have several projects which contain a subset of functionalities from the main babylon app. These apps also have different flavours in terms of UI. 
+
+The labels to define work that is done in each different project are:
+
+- <span style="background-color:#1d76db; color:white; font-weight:600">BUPA 🤕</span> - work in the BUPA app
+- <span style="background-color:#0052cc; color:white; font-weight:600">NHS 👩‍⚕️</span> - work in the GP and hand app
+- <span style="background-color:#9746e2; color:white; font-weight:600">Telus 🇨🇦</span> - work in the Telus app
+- <span style="background-color:#2f2799; color:white; font-weight:600">US 🇺🇸</span> - work in the US app
+
+### <a name="stage"></a>Stage
+Informs in which stage of development the PR is. These labels are used to know which PRs can be reviewed. They are also use to trigger our CI jobs.
+
+- <a name="merge"></a><span style="background-color:#FF7F50; color:black; font-weight:600">Merge</span> - The PR is ready to be merged. Our bot picks up when this label is added. It will then automatically add the PR in the queue to merge back develop. Once merged it waits for our CI system to run all the tests. Only if every test has passed the bot will merge the PR.
```

Comment:
```suggestion
- <a name="merge"></a><span style="background-color:#FF7F50; color:black; font-weight:600">Merge</span> - The PR is ready to be merged. Our bot picks up when this label is added. It will then automatically add the PR in the queue to merge the PRs in order. Once dequeued, it will first merge back develop into the PR, then wait for our CI system to run all the tests. Only if every test has passed the bot will finally merge the PR.
```